### PR TITLE
docs: ensure all placeholders are surrounded by <>

### PR DESCRIPTION
**Pull request by codeliezel**
### Description
This PR adds a <> placeholder for HEROKU_GIT_REMOTE in the documentation.

### Changelog

- [x] `CHANGELOG.md` is updated with user-facing content relevant to this PR. If no changelog is required, then add the `no-changelog-required` label.

### Affected components
- [x] Docs

### Related Issues
https://github.com/hasura/graphql-engine/issues/4906
https://github.com/hasura/graphql-engine-internal/issues/446

### Affected pages
1. https://deploy-preview-5803--hasura-docs.netlify.app/graphql/core/auth/authentication/jwt.html
2. https://deploy-preview-5803--hasura-docs.netlify.app/graphql/core/deployment/deployment-guides/digital-ocean-one-click.html
3. https://deploy-preview-5803--hasura-docs.netlify.app/graphql/core/deployment/deployment-guides/docker.html
4. https://deploy-preview-5803--hasura-docs.netlify.app/graphql/core/migrations/config-v1/advanced/auto-apply-migrations.html
5. https://deploy-preview-5803--hasura-docs.netlify.app/graphql/core/migrations/migrations-setup.html#introduction
1. https://deploy-preview-5803--hasura-docs.netlify.app/graphql/core/deployment/deployment-guides/heroku.html
2. https://deploy-preview-5803--hasura-docs.netlify.app/graphql/core/deployment/deployment-guides/kubernetes.html
3. https://deploy-preview-5803--hasura-docs.netlify.app/graphql/core/deployment/postgres-requirements.html
4. https://deploy-preview-5803--hasura-docs.netlify.app/graphql/core/deployment/enable-https.html

### CODE REVIEW

**Comment by marionschleifer** [comment]
@codeliezel this looks good for this specific example. However, these `<>` need to be added for all placeholder throughout the whole docs. This will give you a nice opportunity to go everything and pick up new tech knowledge 🙂

**Comment by codeliezel** [comment]
> @codeliezel this looks good for this specific example. However, these `<>` need to be added for all placeholders throughout the whole docs. This will give you a nice opportunity to go everything and pick up new tech knowledge

Oh, that is fine. thanks

File: docs/graphql/core/auth/authentication/jwt.rst [hunk 1]
```diff
@@ -553,21 +553,21 @@ Using the flag:
   $ docker run -p 8080:8080 \
       hasura/graphql-engine:latest \
       graphql-engine \
-      --database-url postgres://username:password@hostname:port/dbname \
+      --database-url postgres://<username>:<password>@<hostname>:port/<dbname> \
```
Comment (marionschleifer) [inline docs/graphql/core/auth/authentication/jwt.rst:556]: `port` is also a placeholder.

File: docs/graphql/core/auth/authentication/jwt.rst [hunk 2]
```diff
@@ -553,21 +553,21 @@ Using the flag:
...
+      --admin-secret <myadminsecretkey> \
       --jwt-secret '{"type":"HS256", "key": "3EK6FD+o0+c7tzBNVfjpMkNDi2yARAAKzQlk8O2IKoxQu4nF7EdAh8s3TwpHwrdWT6R"}'
 
 Using env vars:
 
 .. code-block:: shell
 
   $ docker run -p 8080:8080 \
-      -e HASURA_GRAPHQL_ADMIN_SECRET="myadminsecretkey" \
+      -e HASURA_GRAPHQL_ADMIN_SECRET=<myadminsecretkey> \
```
Comment (marionschleifer) [inline docs/graphql/core/auth/authentication/jwt.rst:566]: We should still show quotes: `"<myadminsecretkey>"`

File: docs/graphql/core/auth/authentication/jwt.rst [hunk 3]
```diff
@@ -553,21 +553,21 @@ Using the flag:
...
       -e HASURA_GRAPHQL_JWT_SECRET='{"type":"RS512", "key": "-----BEGIN PUBLIC KEY-----\nMIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQDdlatRjRjogo3WojgGHFHYLugd\nUWAY9iR3fy4arWNA1KoS8kVw33cJibXr8bvwUAUparCwlvdbH6dvEOfou0/gCFQs\nHUfQrSDv+MuSUMAe8jzKE4qW+jK+xQU9a03GUnKHkkle+Q0pX/g6jXZ7r1/xAK5D\no2kQ+X5xK9cipRgEKwIDAQAB\n-----END PUBLIC KEY-----\n"}' \
       hasura/graphql-engine:latest \
       graphql-engine \
-      --database-url postgres://username:password@hostname:port/dbname \
+      --database-url postgres://<username>:<password>@<hostname>:port/<dbname> \
```
Comment (marionschleifer) [inline docs/graphql/core/auth/authentication/jwt.rst:570]: `port` is also a placeholder.

File: docs/graphql/core/deployment/deployment-guides/docker.rst [hunk 4]
```diff
@@ -61,14 +61,14 @@ Edit the ``HASURA_GRAPHQL_DATABASE_URL`` env var value, so that you can connect
 
    #! /bin/bash
    docker run -d -p 8080:8080 \
-     -e HASURA_GRAPHQL_DATABASE_URL=postgres://username:password@hostname:port/dbname \
+     -e HASURA_GRAPHQL_DATABASE_URL=postgres://<username>:<password>@hostname:port/<dbname> \
```
Comment (marionschleifer) [inline docs/graphql/core/deployment/deployment-guides/docker.rst:64]: `port` is also a placeholder.

File: docs/graphql/core/deployment/deployment-guides/docker.rst [hunk 5]
```diff
@@ -61,14 +61,14 @@ Edit the ``HASURA_GRAPHQL_DATABASE_URL`` env var value, so that you can connect
...
      -e HASURA_GRAPHQL_ENABLE_CONSOLE=true \
      hasura/graphql-engine:latest
 
 Examples of ``HASURA_GRAPHQL_DATABASE_URL``:
 
-- ``postgres://admin:password@localhost:5432/my-db``
-- ``postgres://admin:@localhost:5432/my-db`` *(if there is no password)*
+- ``postgres://<admin>:<password>@localhost:5432/<my-db>``
```
Comment (marionschleifer) [inline docs/graphql/core/deployment/deployment-guides/docker.rst:70]: These are specific examples, so we don't need placeholders here.

File: docs/graphql/core/deployment/deployment-guides/docker.rst [hunk 6]
```diff
@@ -100,7 +100,7 @@ command to allow the Docker container to access the host's network:
         :emphasize-lines: 1
 
         docker run -d --net=host \
-          -e HASURA_GRAPHQL_DATABASE_URL=postgres://username:password@hostname:port/dbname \
+          -e HASURA_GRAPHQL_DATABASE_URL=postgres://<username>:<password>@hostname:port/<dbname> \
```
Comment (marionschleifer) [inline docs/graphql/core/deployment/deployment-guides/docker.rst:103]: `port` is also a placeholder.

File: docs/graphql/core/deployment/deployment-guides/docker.rst [hunk 7]
```diff
@@ -114,7 +114,7 @@ command to allow the Docker container to access the host's network:
         :emphasize-lines: 2
 
         docker run -d -p 8080:8080 \
-          -e HASURA_GRAPHQL_DATABASE_URL=postgres://username:password@host.docker.internal:port/dbname \
+          -e HASURA_GRAPHQL_DATABASE_URL=postgres://<username>:<password>@host.docker.internal:port/<dbname> \
```
Comment (marionschleifer) [inline docs/graphql/core/deployment/deployment-guides/docker.rst:117]: `port` is also a placeholder.

File: docs/graphql/core/deployment/deployment-guides/docker.rst [hunk 8]
```diff
@@ -128,7 +128,7 @@ command to allow the Docker container to access the host's network:
         :emphasize-lines: 2
 
         docker run -d -p 8080:8080 \
-          -e HASURA_GRAPHQL_DATABASE_URL=postgres://username:password@docker.for.win.localhost:port/dbname \
+          -e HASURA_GRAPHQL_DATABASE_URL=postgres://<username>:<password>@docker.for.win.localhost:port/<dbname> \
```
Comment (marionschleifer) [inline docs/graphql/core/deployment/deployment-guides/docker.rst:131]: `port` is also a placeholder.

File: docs/graphql/core/deployment/deployment-guides/docker.rst [hunk 9]
```diff
@@ -172,9 +172,9 @@ Run the Docker command with an admin-secret env var
 
     #! /bin/bash
     docker run -d -p 8080:8080 \
-     -e HASURA_GRAPHQL_DATABASE_URL=postgres://username:password@hostname:port/dbname \
+     -e HASURA_GRAPHQL_DATABASE_URL=postgres://<username>:<password>@hostname:port/<dbname> \
```
Comment (marionschleifer) [inline docs/graphql/core/deployment/deployment-guides/docker.rst:175]: `port` is also a placeholder.

File: docs/graphql/core/deployment/deployment-guides/heroku.rst [hunk 10]
```diff
@@ -65,20 +65,20 @@ Step 2: Create an app with **--stack=container**
 ************************************************
 
 Use the `Heroku CLI <https://devcenter.heroku.com/articles/heroku-cli>`__ to create a new Heroku app from inside the cloned directory. Let's call
-the app ``graphql-on-postgres``.
+the app ``todo-hasura1``.
```
Comment (marionschleifer) [inline docs/graphql/core/deployment/deployment-guides/heroku.rst:68]: Why did we change the name here? In the screenshot above it's `graphql-on-postgres`. We should be consistent.

File: docs/graphql/core/deployment/deployment-guides/kubernetes.rst [hunk 11]
```diff
@@ -44,13 +44,13 @@ Edit ``deployment.yaml`` and set the right database url:
   ...
     env:
     - name: HASURA_GRAPHQL_DATABASE_URL
-      value: postgres://username:password@hostname:port/dbname
+      value: postgres://<username>:<password>@hostname:port/<dbname>
```
Comment (marionschleifer) [inline docs/graphql/core/deployment/deployment-guides/kubernetes.rst:47]: `port` is also a placeholder.

File: docs/graphql/core/deployment/deployment-guides/kubernetes.rst [hunk 12]
```diff
@@ -44,13 +44,13 @@ Edit ``deployment.yaml`` and set the right database url:
...
   ...
 
 Examples of ``HASURA_GRAPHQL_DATABASE_URL``:
 
-- ``postgres://admin:password@localhost:5432/my-db``
-- ``postgres://admin:@localhost:5432/my-db`` *(if there is no password)*
+- ``postgres://<admin>:<password>@localhost:5432/<my-db>``
```
Comment (marionschleifer) [inline docs/graphql/core/deployment/deployment-guides/kubernetes.rst:52]: These are specific examples, so we don't need placeholders here.

File: docs/graphql/core/deployment/deployment-guides/kubernetes.rst [hunk 13]
```diff
@@ -119,7 +119,7 @@ Update the ``deployment.yaml`` to set the ``HASURA_GRAPHQL_ADMIN_SECRET`` enviro
         args: ["serve", "--enable-console"]
         env:
         - name: HASURA_GRAPHQL_DATABASE_URL
-          value: postgres://username:password@hostname:port/dbname
+          value: postgres://<username>:<password>@hostname:port/<dbname>
```
Comment (marionschleifer) [inline docs/graphql/core/deployment/deployment-guides/kubernetes.rst:122]: `port` is also a placeholder.

File: docs/graphql/core/deployment/enable-https.rst [hunk 14]
```diff
@@ -36,7 +36,7 @@ Here is a sample ``nginx.conf`` to proxy requests to Hasura:
 
    server {
      listen 80;
-     server_name hasura.my-domain.com;
+     server_name <hasura>.my-domain.com;
```
Comment (marionschleifer) [inline docs/graphql/core/deployment/enable-https.rst:39]: Placeholder should be around `my-domain`, not `hasura`.

File: docs/graphql/core/deployment/enable-https.rst [hunk 15]
```diff
@@ -60,7 +60,7 @@ Here is a sample ``Caddyfile`` to proxy requests to Hasura:
 
 .. code-block:: bash
 
-   hasura.my-domain.com {
+   <hasura>.my-domain.com {
```
Comment (marionschleifer) [inline docs/graphql/core/deployment/enable-https.rst:63]: Placeholder should be around `my-domain`, not `hasura`.

File: docs/graphql/core/deployment/enable-https.rst [hunk 16]
```diff
@@ -69,7 +69,7 @@ The sample ``Caddyfile`` for Caddy 2:
...
```
Comment (marionschleifer) [inline docs/graphql/core/deployment/enable-https.rst:72]: Placeholder should be around `my-domain`, not `hasura`.

File: docs/graphql/core/deployment/postgres-requirements.rst [hunk 17]
```diff
@@ -51,7 +51,7 @@ Here's a sample SQL block that you can run on your database (as a **superuser**)
     -- These permissions/grants are required for Hasura to work properly.
 
     -- create a separate user for hasura
-    CREATE USER hasurauser WITH PASSWORD 'hasurauser';
+    CREATE USER hasurauser WITH PASSWORD hasurauser;
```
Comment (marionschleifer) [inline docs/graphql/core/deployment/postgres-requirements.rst:54]: We need to keep the `''`s here.

File: docs/graphql/core/migrations/config-v1/advanced/auto-apply-migrations.rst [hunk 18]
```diff
@@ -50,7 +50,7 @@ Example:
    # Start Hasura after applying the migrations present in /home/me/my-project/migrations
    docker run -p 8080:8080 \
           -v /home/me/my-project/migrations:/hasura-migrations \
-          -e HASURA_GRAPHQL_DATABASE_URL=postgres://postgres:@postgres:5432/postgres \
+          -e HASURA_GRAPHQL_DATABASE_URL=postgres://<postgres>:@<postgres>:5432/<postgres> \
```
Comment (marionschleifer) [inline docs/graphql/core/migrations/config-v1/advanced/auto-apply-migrations.rst:53]: This is a specific example that users should just be able to copy. We don't need placeholders here.

**Review by marionschleifer — CHANGES_REQUESTED** [review]

File: docs/graphql/core/auth/authentication/jwt.rst (hunk 1 above)
Comment (codeliezel) [reply docs/graphql/core/auth/authentication/jwt.rst:556]: Ok.

File: docs/graphql/core/migrations/config-v1/advanced/auto-apply-migrations.rst (hunk 18 above)
Comment (codeliezel) [reply docs/graphql/core/migrations/config-v1/advanced/auto-apply-migrations.rst:53]: True, I was sceptical too

File: docs/graphql/core/deployment/deployment-guides/docker.rst (hunk 9 above)
Comment (marionschleifer) [reply docs/graphql/core/deployment/deployment-guides/docker.rst:175]: @codeliezel This has not been corrected yet.

File: docs/graphql/core/deployment/deployment-guides/heroku.rst (hunk 10 above)
Comment (marionschleifer) [reply docs/graphql/core/deployment/deployment-guides/heroku.rst:68]: @codeliezel this is not resolved yet. We either have to change the screenshot to reflect the new name (``todo-hasura1``) or change the name back to ``graphql-on-postgres``. Personally, I think the previous name was fine.

File: docs/graphql/core/deployment/deployment-guides/kubernetes.rst (hunk 12 above)
Comment (marionschleifer) [reply docs/graphql/core/deployment/deployment-guides/kubernetes.rst:52]: @codeliezel this is not resolved yet.

**Review by marionschleifer — CHANGES_REQUESTED** [review]

**Review by marionschleifer — APPROVED** [review]

**Review by rikinsk — APPROVED** [review]

**Pull request closed without merging**